docs/sdk/auth/fragments/android/working-with-api.md

```diff
@@ -118,7 +118,27 @@ AWSMobileClient.getInstance().signIn(username, password, null, new Callback<Sign
 });
 ```
 
-## Confirm SignIn
+### Confirm SignIn (MFA)
+
+In order to setup multi-factor authentication, choose `Manual configuration` while setting up auth in the CLI and when you get to the `Multifactor authentication` step, choose these values:
+
+```bash
+ Multifactor authentication (MFA) user login options: ON (Required for all logins, can not be enabled later)
+ For user login, select the MFA types: SMS Text Message
```

Comment:
Idle thought for another day: can we color the prompts different from the input to call into relief what a developer must fill in versus what the questions are?

Reply:
Yeah I'd love to do that - looked into it briefly here and don't see how to offhand without changing the content to maybe have some extra characters. Might be worth doing in the future.